pillar.example

```diff
@@ -5,14 +5,36 @@ apt:
   remove_preferences: true
   clean_preferences_d: true
 
+  apt_conf_d:
```

Comment:
This is a bit of a nitpick... but if this was to be consistent with `preferences` then this should probably be `apt_conf`.

Then again the repos are defined under `repositories` instead of `sources_list`.

Reply:
Or maybe even 'configuration'?